clusters/nerc-ocp-obs/loki/application.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   source:
     repoURL: https://github.com/ocp-on-nerc/nerc-ocp-config.git
     targetRevision: HEAD
-    path: loki/overlays/nerc-ocp-infra
+    path: loki/overlays/nerc-ocp-obs
   destination:
-    name: nerc-ocp-infra
+    name: nerc-ocp-obs
```

Comment:
Let's be sure to keep the infra application for loki here in addition to the obs cluster.